src/Parallel/GlobalCache.hpp

```diff
@@ -277,9 +280,15 @@ class GlobalCache
    *
    * \warning The `function` may be called twice so it should not modify any
    * state in its scope.
+   * \n
```

Comment:
doesn't just a blank line in the comment work? (I haven't tried `\n` in doxygen)

Reply:
No, for `\warning` and `\note`, if you only put a blank line between two `\warning`s back to back, it'll just squash both into a single warning.

Reply:
Okay, sounds good!

Reply:
So `\n` didn't work, but adding `\parblock ... \endparblock` did work.